engine/src/main/java/org/entando/entando/aps/system/services/api/server/ApiRestServer.java

```diff
@@ -283,6 +283,7 @@ else if (token.isExpired()) {
                 String username = token.getLocalUser();
                 IUserManager userManager = (IUserManager) ApsWebApplicationUtils.getBean(SystemConstants.USER_MANAGER, request);
                 UserDetails user = userManager.getUser(username);
+                properties.put(SystemConstants.API_USER_PARAMETER, user);
```

Comment:
The user can be null and in Properies neither the key or the value can be null. 
This line is better after check of user